### PR TITLE
Add certificate endpoints

### PR DESCRIPTION
**Pull request by jboddey**

### CODE REVIEW

**Comment by jhughesbiot** [comment]
Can we get the conflicts resolved before validating this PR?

**Comment by jboddey** [comment]
> Can we get the conflicts resolved before validating this PR?

Merge conflicts have been resolved

File: framework/requirements.txt [hunk 1]
```diff
@@ -22,3 +23,6 @@ pytest-timeout==2.2.0
 
 # Requirements for the report
 markdown==3.5.2
+
+# Requirements for the session
+cryptography==42.0.4
```
Comment (jhughesbiot) [inline framework/requirements.txt:28]: TLS module is at 42.0.7.  Is there any reason to keep this at a lower version?

Comment (jboddey) [reply framework/requirements.txt]: This has now been upgraded.

**Review by jhughesbiot — APPROVED** [review]

**Pull request merged**